client/src/components/ReviewTile.js

```diff
@@ -1,24 +1,109 @@
-import React from "react"
+import React, { useState, useEffect, setStatus } from "react"
+
+import NewReviewForm from "./NewReviewForm.js"
+import EditReviewForm from "./EditReviewForm.js"
 
 const ReviewTile = (props) => {
+  const review = props.review
+
   let upVotes = 0
   let downVotes = 0
-  
-  props.votes.forEach(vote => {
+
+  review.votes.forEach(vote => {
     if(vote.direction === "up") {
       upVotes++
     } else {
       downVotes++
     }
   })
 
-  return (
-    <div>
-      <p>Score: {props.score} out of 5</p>
-      <p>{props.body}</p>
-      <p>Upvotes: {upVotes}  Downvotes: {downVotes}</p>
-    </div>
-  )
+  const reviewId = review.id
+  const showId = review.showId
+ 
+  const handleDelete = async (event) => {
+    event.preventDefault()
+
+    try {
+      const response = await fetch(`/api/v1/shows/${showId}/reviews/${reviewId}`, {
```

Comment:
this url path is too long. We dont need the showId to delete a review. Just the id of the review itself

this is what we want: 
```
/api/v1/reviews/${reviewId}
```

Reply:
disregard other comment. we implemented this change.

Reply:
following up here - I'm still seeing the path as:
```js
fetch(`/api/v1/shows/${showId}/reviews/${reviewId}`
```